odoo/tools/image.py

```diff
@@ -288,7 +288,7 @@ def is_image_size_above(base64_source, size=IMAGE_BIG_SIZE):
     """
     if not base64_source:
         return False
-    if base64_source[:1] == b'P':
+    if base64_source[:1] == b'P' or base64_source[:1] == 'P':
```

Comment:
`in (b'P', 'P')` ? Don't know.

Reply:
Ok for using `in` but we decided to fix this in 12.0 instead too.